### PR TITLE
Mac as device to be used while running from Xcode

### PR DESCRIPTION
**Pull request by kaunteya**
This will help if people who are facing issue https://github.com/vapor/vapor/issues/1681

### CODE REVIEW

File: 3.0/docs/getting-started/hello-world.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ The toolbox will ask you if you'd like to open Xcode automatically, select `yes`
 
 ## Build & Run
 
-You should now have Xcode open and running. Select the [run scheme](xcode.md#run) from the scheme menu,
+You should now have Xcode open and running. Select the [run scheme](xcode.md#run) from the scheme menu and My Mac from devices,
```
Comment (0xTim) [inline 3.0/docs/getting-started/hello-world.md]: Can you change this to `and **My Mac** as the deployment target`?

Comment (kaunteya) [reply 3.0/docs/getting-started/hello-world.md]: @0xTim Done

**Comment by 0xTim** [comment]
Thanks!

**Comment by penny-coin** [comment]
Hey @kaunteya, you just merged a pull request, have a coin! 

You now have 1 coins.

**Pull request merged**